integer_input_array.c

```diff
@@ -0,0 +1,35 @@
+/*Ava N.*/
+/*Given an array of n integers where n > 1, return an array of same size an input array where at every index of the output array should contain the sum of all elements in the array except the element at the given index. You can have it so that the array is input by the user. */
```

Comment:
Nicely done! You did it very efficiently, so great job! The only thing is I would have included a prompt statement so the user knows to input numbers. I wasn't quite sure if you were just stuck in an infinite loop or what.